### PR TITLE
Fix ci fail

### PR DESCRIPTION
**Pull request by renefritze**
I've identified the issue that previously failed the CI. However this needs a custom or modified docker image for Azure, which I'll get to push tomorrow.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -16,7 +16,6 @@ platforms = Linux, Windows, MacOs
 
 
 [options]
-python_requires='>=3'
```
Comment (Czaki) [inline setup.cfg]: Pleas see current version from repository. The problem here are `'`. Without theme everything work fine.

File: setup.py [hunk 2]
```diff
@@ -4,7 +4,8 @@
     long_description = fh.read()
 
 setuptools.setup(
-    install_requires=['sphinx'],
+    install_requires=['sphinx', 'docutils'],
```
Comment (Czaki) [inline setup.py]: also this is in setup.cfg.

**Review by Czaki — CHANGES_REQUESTED** [review]
Why did you use docker image job instead of UsePythonVersion task? (I`m not an expert in azure pipelines)

**Comment by renefritze** [comment]
> Why did you use docker image job instead of UsePythonVersion task? (I`m not an expert in azure pipelines)

Generally I don't like to use the CI Provider's own language ecosystem. Using containers to run your CI in makes it a lot easier to switch Providers, plus I don't need to learn how to manage python (or , c++, etc.) versions in different providers.

**Comment by Czaki** [comment]
> Generally I don't like to use the CI Provider's own language ecosystem. Using containers to run your CI in makes it a lot easier to switch Providers, plus I don't need to learn how to manage python (or , c++, etc.) versions in different providers.

I see. But this create problems with testing application with GUI components. Did you have some script which allow to run GUI test under docker? 

Maybe you need to setup xvfb.

**Comment by renefritze** [comment]
Yes, xvfb works nicely for running anything that needs X from inside the container.

**Comment by renefritze** [comment]
Sorry for the commit spam btw, but the quoting/arg thing in azure is driving me nuts.

**Comment by Czaki** [comment]
No problem.I can squash this PR to single commit.

**Comment by renefritze** [comment]
Could you please update the version on pypi after merging?

**Review by Czaki — APPROVED** [review]

**Comment by Czaki** [comment]
@renefritze uploaded.

**Pull request merged**